Android/app/src/main/java/io/github/project_travel_mate/MainActivity.java

```diff
@@ -233,7 +233,9 @@ private Fragment getFragmentByNavMenuItemId(int id) {
             case R.id.nav_utility:
                 fragment = UtilitiesFragment.newInstance();
                 break;
-
+                //Work In Progress Offline Maps
```

Comment:
is it work in progress or done?

Reply:
Still a work in progress for the MainActivity.java file as I'm still trying to figure out how the feature should be implemented.

Reply:
@Swati4star Hey, just cleaned up this branch for the merging process. Please review it again, thanks!